serializers.py

```diff
@@ -0,0 +1,25 @@
+from marshmallow import Schema, fields
+
+
+class ContragentSchema(Schema):
+    id = fields.Integer()
+    name = fields.Str()
+    account = fields.Int()
+
+
+class PaymentSchema(Schema):
+    id = fields.Integer()
+    contributor = fields.String()
+    amount = fields.Float()
+    period = fields.LocalDateTime()
+    date = fields.Integer()
+    contragent_id = fields.Integer()
+    contract_id = fields.Integer()
+
+
+contragent_schema = ContragentSchema()
```

Comment:
No need to do like this. You can use it as PaymentSchema as example. It will looks like:
```
data, _ = PaymentsSchema().load(json_data)
```

Reply:
changed.